src/Tasks.UnitTests/CodeTaskFactoryTests.cs

```diff
@@ -1040,6 +1043,164 @@ public override bool Execute()
             }
         }
 
+        [Fact]
+        public void EmbedsSourceFileInBinlog()
+        {
+            string sourceFileContent = @"
```

Comment:
This follows the existing style but raw strings are so much better that I think we should put them in all new code (and eventually go back and clean up the existing stuff). Pushing a commit that does that.